### DISCUSSION TITLE
how to use other accounts instead of the burner wallet account?

### DISCUSSION THREAD

**Discussion by mariolim96** [scaffold-eth/scaffold-eth-2#820] [Q&A] [answered]
I'm unable to switch to a default private key generated by Hardhat. 
After I set up the account on Metamask, when I try to connect to Metamask it continues to show the burner wallet key. 
![image](https://github.com/scaffold-eth/scaffold-eth-2/assets/37416359/d0f77922-4743-4694-a4d4-7c6f9b7007a4)
In the scaffold setting onlyLocalBurnerWallet is setted to false

How could I use other accounts when the chain is 31337 or how could I setup a hardhat account into the burner wallet?

**Comment by carletex** [comment]
Hey @mariolim96 not sure If I understand the question.

A random generated burner wallets load by default (pk stored in localStorage). You can disconnect (clicking the burner wallet address + disconnect) and connect any wallet 

![image](https://github.com/scaffold-eth/scaffold-eth-2/assets/2486142/0c3bca14-b236-47aa-b5bd-073ab7b625f5)

E.g. if you connect metamask, that would be the wallet that the dApp will use.

> In the scaffold setting onlyLocalBurnerWallet is setted to false

That setting is to enable / disable burner wallet in live networks (in the local network is always enabled)

> How could I use other accounts when the chain is 31337 or how could I setup a hardhat account into the burner wallet?

You can use any wallet / account you want. Also, keep in mind that you can use the "Faucet" to get funds in any connected wallet.

**Reply by mariolim96** [reply, marked as answer]
I was missing a phase, after selected metamask i had to click on this icon  and click connect.
![image](https://github.com/scaffold-eth/scaffold-eth-2/assets/37416359/03a2faed-0db6-49ef-bb93-0379c1356dec)

Thanks a lot :)
